src/StaffWebUI/Components/Pages/Tickets/TicketsFilter.razor.css

```diff
@@ -3,7 +3,7 @@
     position: sticky;
     top: var(--header-height);
     background: var(--neutral-fill-stealth-active);
-    z-index: 2;
+    z-index: 4;
```

Comment:
Is this needed due to a change in the FluentUI packages? Fine if so, just want to understand.

Reply:
This was for the sticky header to appear below the filter when using the `FluentDataGrid`